components/toast.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Docs</title>
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/fontawesome.min.css" integrity="sha512-OdEXQYCOldjqUEsuMKsZRj93Ht23QRlhIb8E/X0sbwZhme8eUw6g8q7AdxGJKakcBbv7+/PX0Gc2btf7Ru8cZA==" crossorigin="anonymous" /> -->
+    
+    <link href="/main.css" rel="stylesheet" />
+    <link href="/css/toast.css" rel="stylesheet" />
+    <link href="/css/button.css" rel="stylesheet" />
+    
+</head>
+<body>
+    <div id="page">
```

Comment:
Hi Sparsh, any reason why you are using id to refrence the div on all pages instead of class?

Reply:
Nothing specific , just used it . Should I change it ?

Reply:
Yes u should. Id selectors are highly specific . Use them only when required.

Reply:
okay , thanks